src/oscar/templatetags/category_tags.py

```diff
@@ -67,8 +67,8 @@ def __iter__(self):
         yield self
 
 
-@register.simple_tag(name="category_tree")   # noqa: C901 too complex
-def get_annotated_list(depth=None, parent=None):
+@register.simple_tag(name="category_tree")
+def get_annotated_list(depth=None, parent=None):    # noqa: C901 too complex
```

Comment:
This is because of switching to Python 3.8, eh? I could not find it documented anywhere.

Reply:
Yeah - I couldn't find anything either but how isort checks decorated functions seems to change in Python 3.8 .

Reply:
flake8, rather